ci/jenkins/Jenkinsfile.j2

```diff
@@ -116,8 +116,16 @@ if (rebuild_docker_images) {
 
 lint()
 
-build()
+build('Build')
 
-test()
+test('Test')
+
+if (rebuild_docker_images) {
```

Comment:
so does this build/test twice then? i guess that kinda makes sense but it's going to take a long time

Reply:
Right it runs it twice (once with the old images, once with the new), we can keep the time down if we stay on top of sharding